catalog/core/catalog-core-commands/src/test/groovy/org/codice/ddf/commands/catalog/ExportCommandSpec.groovy

```diff
@@ -0,0 +1,283 @@
+package org.codice.ddf.commands.catalog
+
+import ddf.catalog.CatalogFramework
+import ddf.catalog.data.BinaryContent
+import ddf.catalog.data.Metacard
+import ddf.catalog.data.impl.AttributeImpl
+import ddf.catalog.data.impl.MetacardImpl
+import ddf.catalog.data.impl.ResultImpl
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder
+import ddf.catalog.operation.QueryRequest
+import ddf.catalog.operation.ResourceRequest
+import ddf.catalog.operation.impl.QueryResponseImpl
+import ddf.catalog.operation.impl.ResourceResponseImpl
+import ddf.catalog.resource.ResourceNotFoundException
+import ddf.catalog.resource.impl.ResourceImpl
+import ddf.catalog.transform.MetacardTransformer
+import groovy.xml.MarkupBuilder
+import org.apache.karaf.shell.api.console.Session
+import org.osgi.framework.BundleContext
+import org.osgi.framework.ServiceReference
+
+import javax.activation.MimeType
+import java.nio.file.Paths
+import java.util.zip.ZipFile
+
+class ExportCommandSpec extends spock.lang.Specification {
+
+    ExportCommand exportCommand
+
+    CatalogFramework catalogFramework
+
+    MetacardTransformer xmlTransformer
+
+    File tmpHomeDir
+
+    void setup() {
+        tmpHomeDir = File.createTempDir()
+        System.setProperty("ddf.home", tmpHomeDir.canonicalPath)
+
+        ServiceReference xmlTransformerReference = Mock(ServiceReference)
+
+        xmlTransformer = Mock(MetacardTransformer)
+
+        xmlTransformer.transform(_ as Metacard, _ as Map) >> { metacard, map ->
+            metacardToXmlTransformer(metacard, map)
+        }
+
+        BundleContext bundleContext = Mock(BundleContext) {
+            getServiceReferences(MetacardTransformer, '(id=xml)') >> [xmlTransformerReference]
+            getService(xmlTransformerReference) >> xmlTransformer
+        }
+
+        catalogFramework = Mock(CatalogFramework)
+        catalogFramework.query(_ as QueryRequest) >> { QueryRequest req ->
+            new QueryResponseImpl(req, [], 0)
+        }
+        catalogFramework.getLocalResource(_ as ResourceRequest) >> {
+            throw new ResourceNotFoundException('Could not find exception')
+        }
+
+        exportCommand = new ExportCommand(filterBuilder: new GeotoolsFilterBuilder(),
+                bundleContext: bundleContext, catalogFramework: catalogFramework)
+
+    }
+
+    void cleanup() {
+        assert tmpHomeDir?.deleteDir()
```

Comment:
`deleteDir` returns true if the delete succeeds (or the folder doesn't exist) and false if it fails (or isn't a directory). This way ensure each test is cleaned up and have no false negative failures because state that wasn't cleaned up... Ironically it was unable to delete the directory every time causing failures... removed the assert so we best effort delete (but still temp file so should get cleaned up)